test/functional/walletbackup.py

```diff
@@ -190,6 +190,22 @@ def run_test(self):
         assert_equal(self.nodes[1].getbalance(), balance1)
         assert_equal(self.nodes[2].getbalance(), balance2)
 
+        # backup to file self must fail
+        try:
```

Comment:
You can replace these entire try:except: blocks with a simple assert_raises_jsonrpc call (which also checks the error number).